apps/dh/api-dh/source/DataHub.WebApi/Clients/ElectricityMarket/V1/nswag.json

```diff
@@ -3,7 +3,7 @@
   "defaultVariables": null,
   "documentGenerator": {
     "fromDocument": {
-      "url": "https://localhost:6005/swagger/v1/swagger.json",
+      "url": "https://localhost:7085/swagger/v1/swagger.json",
```

Comment:
Suggestion: Maybe we should point it at dev02?

Reply:
Fixed